spec/system/publishers_can_publish_a_vacancy_as_a_school_spec.rb

```diff
@@ -141,14 +141,14 @@
       fill_in_important_dates_fields(vacancy)
       click_on I18n.t("buttons.continue")
 
-      click_on I18n.t("buttons.continue")
-
       fill_in_applying_for_the_job_form_fields(vacancy)
       click_on I18n.t("buttons.continue")
 
       fill_in_applying_for_the_job_details_form_fields(vacancy)
       click_on I18n.t("buttons.continue")
 
+      click_on I18n.t("buttons.continue")
+
```

Comment:
Can't figure out why we click continue twice here?

Reply:
We fill in the job details form fields, then press continue on that page and at that point we are at the supporting documents step - we then click continue again (without uploading anything) to go to the next step